tfinstall/gitref/git_ref.go

```diff
@@ -12,25 +12,27 @@ import (
 
 	"github.com/go-git/go-git/v5"
 	"github.com/go-git/go-git/v5/plumbing"
+
+	"github.com/hashicorp/terraform-exec/tfinstall"
 )
 
-type GitRefOption struct {
+type Option struct {
 	installDir string
 	repoURL    string
 	ref        string
 }
 
-var _ ExecPathFinder = &GitRefOption{}
+var _ tfinstall.ExecPathFinder = &Option{}
```

Comment:
Nitpick: If we move this to a test file then we can avoid the runtime dependency on `tfinstall`.